texar/torch/data/data/data_iterators.py

```diff
@@ -95,6 +94,7 @@ def _move_fn(x):
     from texar.torch.data.data.data_iterators_utils import \
         TexarMultiProcessingDataLoaderIter as _MultiProcessingDataLoaderIter
 
+
```

Comment:
These whitespace changes are probably why the CI lint failed. Let's get rid of these and only have changes relevant to the issue in this PR.

Reply:
Wow, thanks for the advice.